dev/src/__tests__/jest_testing.md

```diff
@@ -0,0 +1,57 @@
+# How to run jest tests
```

Comment:
Note that I revised this file and moved it to top-level `doc` folder: `doc/jest-testing.md`. We want to keep docs in one place, otherwise it becomes a bit difficult to find...

Reply:
Please check that the revised version is okay, or update it with any necessary changes, and delete this file...